File: Python/1015.py
import math

one = str(input()).split(" ")
two = str(input()).split(" ")

x1 = float(one[0])
x2 = float(two[0])
y1 = float(one[1])
y2 = float(two[1])

distance = ((x2-x1)**2) + ((y2-y1)**2)
distance = math.sqrt(distance)


print("{:.4f}".format(distance))

File: JavaScript/1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
let pontOne = lines.shift().split(' ');
let pontTwo = lines.shift().split(' ');

let x1 = parseFloat(pontOne[0]);
let y1 = parseFloat(pontOne[1]);
let x2 = parseFloat(pontTwo[0]);
let y2 = parseFloat(pontTwo[1]);

let distance = Math.sqrt( Math.pow(x2-x1, 2) + Math.pow(y2-y1, 2));
console.log(distance.toFixed(4));